frontend_app_kanban/src/component/Kanban.jsx

```diff
@@ -146,13 +155,20 @@ export class Kanban extends React.Component {
   }
 
   handleCardEdited = (card) => {
-    this.setState(prevState => ({
-      editedCardInfos: null,
-      board: card.id
-        ? changeCard(prevState.board, card.id, card)
-        : addCard(prevState.board, prevState.editedCardInfos.column, { ...card, id: uuidv4() }),
-      saveRequired: true
-    }))
+    this.setState(prevState => {
+      const newBoard = (
+        card.id
+          ? changeCard(prevState.board, card.id, card)
+          : addCard(prevState.board, prevState.editedCardInfos.column, { ...card, id: uuidv4() })
+      )
+
+      this.save(newBoard)
```

Comment:
I thought the same, but I haven't found any way to avoid this after your recommendation to remove `saveRequired` and to not handle saving the board in componentDidUpdate. 

We retrieve the current board in prevState and update it according to the action that was done. We then need to save the new state. We can't use state.board directly because there could have been a call to setState just before. That's why a function is used in this setState. This is because we don't have guaranties react-kanban don't call several of these handlers for any unique modification.

If you have any suggestion we can discuss it. @Skylsmoi

Reply:
I've restored saveRequired. Review please @Skylsmoi